tests/wallet-automation/typhon/typhon-wallet-registration.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { test, chromium } from '@playwright/test';
+import { getWalletCredentials } from './credentials';
+import { getSeedPhrase } from './seed-phrase';
+const path = require('path');
+
+test('import wallet', async ({ }) => {
+  const extensionPath: string = path.resolve(__dirname, 'extensions');
```

Comment:
The unzip-crx-3 will extract the extension in `extensions/{typhonid}` So you need to change this line to match that:

 ` const extensionPath: string = path.join(__dirname, '../extensions/KFDNIEFADAANBJODLDOHAEDPHAFOFFOH_3_1_2_0');`

Reply:
const extensionPath: string = path.join(__dirname, '../extensions/KFDNIEFADAANBJODLDOHAEDPHAFOFFOH_3_1_2_0');
 
the file name in the extensions folder is dynamic from this "_3_1_2_0" and onwards therefore I don't think it's necessary to make the ID dynamic